components/brave_wallet/browser/BUILD.gn

```diff
@@ -309,6 +309,7 @@ source_set("utils") {
     "//brave/components/brave_wallet/common:mojom",
     "//brave/components/version_info",
     "//brave/third_party/bip39wally-core-native:bip39wally-core",
+    "//components/base32:base32",
```

Comment:
When ConvertFEVMtoFVM is moved to FilAddress there should be no changes in gn or DEPS files